apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
+                'You\'ve been flagged as spam, sorry about that.')
```

Comment:
This string should get passed through `_` like in [line 157](https://github.com/EnTeQuAk/olympia/blob/feature/add-honeypot-to-collections-add/apps/bandwagon/forms.py#L157).